### PR TITLE
[stable/mysql] add namespace metadata

### PR DESCRIPTION
**Pull request by eh-am**
Signed-off-by: eduardo aleixo <eduardoaleixomartins@gmail.com>

#### What this PR does / why we need it:
this is needed for when helm is used purely as a templating tool,
since helm template does not add the namespace
https://github.com/helm/helm/issues/3553

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/mysql/templates/configurationFiles-configmap.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "mysql.fullname" . }}-configuration
+  namespace: {{ .Release.Namespace }}
```
Comment (kfox1111) [inline stable/mysql/templates/configurationFiles-configmap.yaml:6]: doesnt helm do this already?

Comment (eh-am) [reply stable/mysql/templates/configurationFiles-configmap.yaml:6]: only at deployment time, so if you are using helm purely as a template tool (for instance in Spinnaker) it doesn't do that.

**Comment by kfox1111** [comment]
ah. ok.

**Pull request merged**